brave-apache-http-interceptors/pom.xml

```diff
@@ -42,5 +42,10 @@
         <artifactId>brave-http-tests</artifactId>
         <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.apache.httpcomponents</groupId>
+          <artifactId>httpasyncclient</artifactId>
+          <scope>test</scope>
```

Comment:
build seems to break on a snapshot repo https://github.com/openzipkin/brave/issues/309

wonder if it is actually only this PR. can you try adding the version either here as `<version>${httpcomponents.version}</version>` or doing the same in the root pom under dependency management?